spec/rubocop/cop/rspec/aggregate_examples_spec.rb

```diff
@@ -0,0 +1,790 @@
+RSpec.describe RuboCop::Cop::RSpec::AggregateExamples, :config do
+  subject(:cop) { described_class.new(config) }
+
+  let(:cop_config) { {} }
+
+  shared_examples 'detects and autocorrects' do |offensive_source, good_source|
+    it 'does not detect an offense in good_source code' do
+      expect_no_offenses(good_source)
+    end
+
+    it 'detects an offense in offensive source code' do
+      expect_offense(offensive_source)
+    end
+
+    bad_code_without_message =
+      offensive_source.lines.delete_if { |line| line =~ /\A\s*\^/ }.join
+    include_examples 'autocorrect', bad_code_without_message, good_source
+  end
+
+  shared_examples 'detects, but does not autocorrect' do |offensive_source|
+    it 'detects an offense in offensive source code code' do
+      expect_offense(offensive_source)
+    end
+
+    it 'does not autorcorrect source code' do
+      bad_code_without_message =
+        offensive_source.lines.delete_if { |line| line =~ /\A\s*\^/ }.join
+      expect(autocorrect_source(bad_code_without_message, 'spec/foo_spec.rb'))
+        .to eq(bad_code_without_message)
+    end
+  end
```

Comment:
Should I extract those two `shared_example`'s somewhere? Will they be useful elsewhere?

Reply:
`expect_no_corrections` will help with this:
```rb
  shared_examples 'detects, but does not autocorrect' do |offensive_source|
    it 'detects an offense in offensive source code code' do
      expect_offense(offensive_source)
      expect_no_corrections
    end
  end
```

Reply:
Awesome, thank you! 👍